Source/Scene/Cesium3DTileStyle.js

```diff
@@ -0,0 +1,442 @@
+    var DEFAULT_JSON_BOOLEAN_EXPRESSION = true;
+    var DEFAULT_JSON_NUMBER_EXPRESSION = 1.0;
+
+    /**
+     * A style that is applied to a {@link Cesium3DTileset}.
+     * <p>
+     * Evaluates an expression defined using the
+     * {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/Styling|3D Tiles Styling language}.
+     * </p>
+     *
+     * @alias Cesium3DTileStyle
+     * @constructor
+     *
+     * @param {String|Object} [style] The url of a style or an object defining a style.
+     *
+     * @example
+     * tileset.style = new Cesium.Cesium3DTileStyle({
+     *     color : {
+     *         conditions : [
+     *             ['${Height} >= 100', 'color("purple", 0.5)'],
+     *             ['${Height} >= 50', 'color("red")'],
+     *             ['true', 'color("blue")']
+     *         ]
+     *     },
+     *     show : '${Height} > 0',
+     *     meta : {
+     *         description : '"Building id ${id} has height ${Height}."'
+     *     }
+     * });
+     *
+     * @example
+     * tileset.style = new Cesium.Cesium3DTileStyle({
+     *     color : 'vec4(${Temperature})',
+     *     pointSize : '${Temperature} * 2.0'
+     * });
+     *
+     * @see {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/Styling|3D Tiles Styling language}
+     */
+    function Cesium3DTileStyle(style) {
+        this._style = undefined;
+        this._ready = false;
+        this._color = undefined;
+        this._show = undefined;
+        this._pointSize = undefined;
+        this._meta = undefined;
+
+        this._colorShaderFunction = undefined;
+        this._showShaderFunction = undefined;
+        this._pointSizeShaderFunction = undefined;
+        this._colorShaderFunctionReady = false;
+        this._showShaderFunctionReady = false;
+        this._pointSizeShaderFunctionReady = false;
+
+        var promise;
+        if (typeof style === 'string') {
+            promise = loadJson(style);
+        } else {
+            promise = when.resolve(style);
+        }
+
+        var that = this;
+        this._readyPromise = promise.then(function(styleJson) {
+            setup(that, styleJson);
+            return that;
+        });
+    }
+
+    function setup(that, styleJson) {
+        that._style = clone(styleJson, true);
+
+        styleJson = defaultValue(styleJson, defaultValue.EMPTY_OBJECT);
+
+        that._colorShaderFunctionReady = !defined(styleJson.color);
+        that._showShaderFunctionReady = !defined(styleJson.show);
+        that._pointSizeShaderFunctionReady = !defined(styleJson.pointSize);
+
+        var colorExpression = defaultValue(styleJson.color, DEFAULT_JSON_COLOR_EXPRESSION);
+        var showExpression = defaultValue(styleJson.show, DEFAULT_JSON_BOOLEAN_EXPRESSION);
+        var pointSizeExpression = defaultValue(styleJson.pointSize, DEFAULT_JSON_NUMBER_EXPRESSION);
+
+        var expressions = styleJson.expressions;
```

Comment:
I think this should be renamed from `expressions` to `defines` before this 3DTile will become in stable branch to prevent a spec update in next release.